scripts/v8-to-v9.py

```
@@ -0,0 +1,178 @@
+
+    # since these states remain unchanged, we can just copy them.
+    # chain params -> copy
+    blank_data['initial_height'] = data['initial_height']
+    blank_data['consensus_params'] = data['consensus_params']
+    blank_data['chain_id'] = chain_id
+
+    # x/assets -> copy
+    blank_data['app_state']['assets'] = data['app_state']['assets']
+
+    # x/auth -> copy
+    blank_data['app_state']['auth']= data['app_state']['auth']
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)